lighthouse-core/test/report/html/renderer/performance-category-renderer-test.js

```diff
@@ -154,12 +154,17 @@ describe('PerfCategoryRenderer', () => {
 
   it('renders the failing diagnostics', () => {
     const categoryDOM = renderer.render(category, sampleResults.categoryGroups);
-    const diagnosticSection = categoryDOM.querySelectorAll('.lh-category > .lh-audit-group')[3];
```

Comment:
this just needed to have `s/3/2` done, but this is a better version of the test :)